src/api/spec/features/webui/users/users_home_project_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'browser_helper'
 
 RSpec.feature "User's home project creation", type: :feature, js: true do
-  let!(:user) { create(:confirmed_user, login: 'moi') }
+  let!(:user) { create(:confirmed_user, login: 'hans') }
```

Comment:
why is this needed?

Reply:
Because 'moi' was already used which lead to `Project.restore` kicking in.